test/GPv2Settlement.test.ts

```diff
@@ -206,7 +207,7 @@ describe("GPv2Settlement", () => {
       }
 
       const [inTransfers, outTransfers] = parseTransfers(
-        await settlement.computeTradeExecutionsTest(
+        await settlement.callStatic.computeTradeExecutionsTest(
```

Comment:
Why making the call static? The state of the test chain should be reset before any test anyway.

Reply:
So we get return values instead of a transaction receipt.